docs/source/accelerators/mps_basic.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+.. _mps_basic:
+
+MPS training (basic)
```

Comment:
I would suggest Apple or M1 in the title to make it easier for users to discover it.

Reply:
done :)